CMakeLists.txt

```diff
@@ -175,6 +175,10 @@ IF(NOT MSVC)
     GLOBAL_APPEND(KOKKOS_LINK_OPTIONS -DKOKKOS_DEPENDENCE)
 ENDIF()
 
+IF(Kokkos_ENABLE_TESTS)
+  find_package(GTest)
```

Comment:
Is there a way to check whether the external gtest has pthread?

Reply:
As discussed, gtest doesn't export this information.

On my machine the best I can do (assuming static library) is to use `nm`:
```
nm -go /usr/lib/x86_64-linux-gnu/libgtest.a | grep pthread
/usr/lib/x86_64-linux-gnu/libgtest.a:gtest-all.cc.o:                 U pthread_getspecific
/usr/lib/x86_64-linux-gnu/libgtest.a:gtest-all.cc.o:                 w __pthread_key_create
/usr/lib/x86_64-linux-gnu/libgtest.a:gtest-all.cc.o:                 U pthread_key_create
/usr/lib/x86_64-linux-gnu/libgtest.a:gtest-all.cc.o:                 U pthread_key_delete
/usr/lib/x86_64-linux-gnu/libgtest.a:gtest-all.cc.o:                 U pthread_mutex_destroy
/usr/lib/x86_64-linux-gnu/libgtest.a:gtest-all.cc.o:                 U pthread_mutex_init
/usr/lib/x86_64-linux-gnu/libgtest.a:gtest-all.cc.o:                 U pthread_mutex_lock
/usr/lib/x86_64-linux-gnu/libgtest.a:gtest-all.cc.o:                 U pthread_mutex_unlock
/usr/lib/x86_64-linux-gnu/libgtest.a:gtest-all.cc.o:                 U pthread_self
/usr/lib/x86_64-linux-gnu/libgtest.a:gtest-all.cc.o:                 U pthread_setspecific
```